run-ut.sh

```diff
@@ -141,7 +141,7 @@ test_files=`find ${STARROCKS_TEST_BINARY_DIR} -type f -perm -111 -name "*test" |
 # run cases in starrocks_test in parallel if has gtest-parallel script.
 # reference: https://github.com/google/gtest-parallel
 if [ -x ${GTEST_PARALLEL} ]; then
-    ${GTEST_PARALLEL} ${STARROCKS_TEST_BINARY_DIR}/starrocks_test --serialize_test_cases ${GTEST_PARALLEL_OPTIONS}
+    ${GTEST_PARALLEL} ${STARROCKS_TEST_BINARY_DIR}/starrocks_test --serialize_test_cases ${GTEST_PARALLEL_OPTIONS} --gtest_filter="StringFunctionLocateTest*"
```

Comment:
why?

Reply:
This modification was used to debug unit test, and is already removed now.